cli/tests/integration_tests.rs

```diff
@@ -1294,7 +1296,11 @@ fn bundle_watch_fail() {
   std::thread::sleep(std::time::Duration::from_secs(1));
   assert!(stderr_lines.next().unwrap().contains("file_to_watch.js"));
   assert!(stderr_lines.next().unwrap().contains("error:"));
-  assert!(!deno.wait().unwrap().success());
+  assert!(stderr_lines.next().unwrap().contains("Bundle failed!"));
+  // the target file hasn't been created yet
+  assert!(!target_file.is_file());
+  // Make sure the Deno's process is still alive
+  assert!(deno.try_wait().unwrap().is_none());
 
```

Comment:
Nice! Could you create a file and check that watcher actually restarts as expected?

Reply:
Sure, done it 👍